style.css

```diff
@@ -0,0 +1,212 @@
+.main-container {
+  display: grid;
+  margin: 0 auto;
+  grid-template-areas:
+    "top"
+    "middle"
+    "bottom";
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto;
+  width: 75%;
+  font-family: "Raleway", sans-serif;
+}
+
+/* Top Section */
+
+.top {
+  grid: top;
+  display: flex;
+  flex-direction: column;
+  background-image: url("https://web.archive.org/web/20140226220919im_/http://images.apple.com/v/home/as/images/your_verse_hero_2x.jpg");
+  background-repeat: no-repeat;
+  background-position: 50% 100%;
+  background-origin: padding-box;
+  background-size: 100%;
+}
+
+/* Nav Bar */
+
+.nav {
+  margin: 80px auto 120px auto;
+  width: 70%;
+  display: grid;
+  grid-template-columns: repeat(9, 1fr) auto;
+  background:
+    -webkit-gradient(
+      linear,
+      0 0,
+      0 100%,
+      from(rgba(0, 0, 0, 0.2)),
+      color-stop(0.05, rgba(0, 0, 0, 0)),
+      color-stop(0.97, rgba(0, 0, 0, 0)),
+      to(rgba(0, 0, 0, 0.45))
+    ),
+    -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.2))),
+    -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+  align-items: center;
+  justify-items: center;
+  border-radius: 4px;
+  border: 1px solid #808080;
+}
+
+.nav a {
+  color: white;
+  font-size: 16px;
+  text-decoration: none;
+  text-align: center;
+  justify-self: stretch;
+  align-self: stretch;
+  border-right: 1px solid #808080;
+  padding: 8px 0;
+}
+
+.bottom a {
+  color: rgb(0, 136, 204);
+  text-decoration: none;
+}
+
+.nav a:hover {
+  background-color: #3a3a3a;
+}
+
+.nav div {
+  display: flex;
+  position: relative;
+}
+
+.nav div input {
+  background: #707070;
+  outline: none;
+  border-top-style: none;
+  border-right-style: none;
+  border-left-style: none;
+  border-bottom-style: none;
+  padding-left: 10px;
+  order: 2;
+}
+
+.line-3 svg {
+  padding-left: 10px;
+}
+
+.nav div svg {
+  /* position: absolute;
+  top: 2px;
+  left: -12px; */
```

Comment:
# [REQUIRED] - REMOVE - Commented Out Code
- ***Commented Out Code*** need not be part of your final codes. You can use git history to track for changes and review previous version of your code. Kindly remove all the [commented code](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) in your files including this one. 👆